we build krita with qt5, so this has to match

diff --git a/gmic-qt/CMakeLists.txt b/gmic-qt/CMakeLists.txt
index d0fb719..c106470 100644
--- a/gmic-qt/CMakeLists.txt
+++ b/gmic-qt/CMakeLists.txt
@@ -156,3 +156,3 @@ find_package(Threads REQUIRED)
 #
-find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core)
+find_package(QT NAMES Qt5 REQUIRED COMPONENTS Core)
 find_package(Qt${QT_VERSION_MAJOR} ${MIN_QT_VERSION}
